jiant/pytorch_transformers_interface/modules.py

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
I don't know TXL/GPT-2 that well, so make sure you understand the ways that they're different from BERT. It'll be easy to add subtle bugs here that hurt performance slightly. Take a close look at the model papers and the pytorch_transformers code.